src/components/forms/controls/DatePicker/DatePicker.module.scss

```diff
@@ -0,0 +1,213 @@
+  opacity: 0.7;
+}
+:global(.react-datepicker__navigation-icon) {
+  // original icon
+  display: none;
+}
+
+:global(.react-datepicker__day-names) {
+  border-bottom: 0.1rem solid bk.$theme-date-picker-rule-default;
+}
+
+:global(.react-datepicker__day-name),
+:global(.react-datepicker__day) {
+  display: inline-block;
+  width: 30px;
+  line-height: 30px;
+  margin: 5px 0;
+  color: bk.$theme-date-picker-text-default;
+}
+
+:global(.react-datepicker__day--today) {
+  font-weight: bk.$font-weight-semibold;
+}
+
+:global(.react-datepicker__day--outside-month),
+:global(.react-datepicker__day--disabled) {
+  color: bk.$theme-date-picker-text-disabled;
+}
+
+:global(.react-datepicker__day--outside-month.react-datepicker__day--disabled) {
+  color: bk.$theme-date-picker-text-unavailable;
+}
+
+:global(.react-datepicker__day--selected),
+:global(.react-datepicker__day--keyboard-selected) {
+  background: bk.$theme-date-picker-background-selected;
+  border-radius: bk.$border-radius-circle;
+  color: bk.$theme-date-picker-text-selected;
+}
+
+:global(.react-datepicker__day:not([aria-disabled="true"]):hover),
+:global(.react-datepicker__day--selected:not([aria-disabled="true"]):hover),
+:global(.react-datepicker__day--keyboard-selected:not([aria-disabled="true"]):hover) {
+  background: bk.$theme-date-picker-background-hover;
+  border-radius: bk.$border-radius-circle;
+  color: bk.$theme-date-picker-text-hover;
+}
+
+:global(.react-datepicker__day--in-range),
+:global(.react-datepicker__day--in-selecting-range:not(
+  .react-datepicker__month-text--in-range,
+  .react-datepicker__quarter-text--in-range,
+  .react-datepicker__year-text--in-range
+)) {
+  background: bk.$theme-date-picker-background-selected;
+  border-radius: 0;
+  color: bk.$theme-date-picker-text-selected;
+}
+:global(.react-datepicker__day--in-range:not([aria-disabled="true"]):hover),
+:global(.react-datepicker__day--in-selecting-range:not([aria-disabled="true"]):hover) {
+  background: bk.$theme-date-picker-background-hover;
+  border-radius: 0;
+  color: bk.$theme-date-picker-text-hover;
+}
+
+:global(.react-datepicker__day--range-start),
+:global(.react-datepicker__day--selecting-range-start),
+:global(.react-datepicker__day--range-start:not([aria-disabled="true"]):hover) {
+  border-radius: bk.$border-radius-circle 0 0 bk.$border-radius-circle;
+}
+
+:global(.react-datepicker__day--range-end),
+:global(.react-datepicker__day--selecting-range-end),
+:global(.react-datepicker__day--range-end:not([aria-disabled="true"]):hover) {
+  border-radius: 0 bk.$border-radius-circle bk.$border-radius-circle 0;
+}
+
+:global(.react-datepicker__day--range-start.react-datepicker__day--range-end),
+:global(.react-datepicker__day--range-start.react-datepicker__day--range-end:not([aria-disabled="true"]):hover) {
+  border-radius: bk.$border-radius-circle;
+}
+
+/* stylelint-disable-next-line */
+:global(.react-datepicker__day:not(&--selected):hover) {
```

Comment:
We shouldn't be using Sass-style string concatenation here (`&--selected`), with native CSS nesting this is no longer supported. Can instead write `.react-datepicker__day--selected` in full.